doc/classes/TextServer.xml

```diff
@@ -1896,10 +1896,13 @@
 			BiDi override for lists.
 			Structured text options: list separator [code]String[/code].
 		</constant>
+		<constant name="STRUCTURED_TEXT_GDSCRIPT" value="6" enum="StructuredTextParser">
+			BiDi override for GDScript.
+		</constant>
 		<constant name="STRUCTURED_TEXT_NONE" value="5" enum="StructuredTextParser">
 			Use default Unicode BiDi algorithm.
 		</constant>
```

Comment:
It would be good to keep them in ascending order, so either GDSCRIPT should get value 5, or it should come after NONE. (I guess the former if NONE is meant to be the last before CUSTOM?).

Reply:
BTW, is it GDScript specific, or would it also be relevant for e.g. editing C#, JSON, or other languages/files in the builtin script editor?

Reply:
Fixed, also removed redundant `STRUCTURED_TEXT_NONE` which is same as `STRUCTURED_TEXT_DEFAULT` in all case. Nothing depends on this enum order, the only special option is `CUSTOM` (calls `_structured_text_parser` GDScript method instead of built-in parser).